server/controllers/feeds/feeds.coffee

```diff
@@ -0,0 +1,27 @@
+CONFIG = require 'config'
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+root = CONFIG.fullPublicHost()
+items_ = __.require 'controllers', 'items/lib/items'
+error_ = __.require 'lib', 'error/error'
+RSS = require 'rss'
+limit = 100
+
+module.exports =
+  get: (req, res, next)->
+    feed = new RSS({"title":"New Public Items"})
+    { query } = req
+    items_.publicByDate limit
+    .then (items) -> rssSerializer(feed, items)
+    .then -> res.send(feed.xml())
+    .catch error_.Handler(req, res)
+
+rssSerializer = (feed, items)->
+  items.map (item)->
+    title: item["title"]
+    guid: item["entity"]
```

Comment:
the item guid should be `item._id` (which is a CouchDB uuid), not it's `item.entity` which will be the same on all items from the same entity